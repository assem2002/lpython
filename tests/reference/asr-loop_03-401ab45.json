{
    "basename": "asr-loop_03-401ab45",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/loop_03.py",
    "infile_hash": "0dce15506e10f77faf350e3e7bf5dc4f39a6723c74f33eeb1f6342bc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-loop_03-401ab45.stderr",
    "stderr_hash": "59d41371ca01bf32c55d25b659c2c8c03947ef8396a96c4c560b9adb",
    "returncode": 2
}