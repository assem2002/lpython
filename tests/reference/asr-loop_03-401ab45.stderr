semantic error: Assignment to loop variable `i` is not allowed
 --> tests/errors/loop_03.py:5:9
  |
5 |         i=i*10
  |         ^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
